### PR TITLE
JS2 Week3 HomeWork

### PR DESCRIPTION
**Pull request by hamoudi-hamoudi**

### CODE REVIEW

File: javascript/javascript2/week3/Warmup/warmUp.js [hunk 1]
```diff
@@ -0,0 +1,116 @@
...
+function doubleClickDetector() {
+  click++;
+  if (click === 1) {
+    setTimeout(() => {
+      click = 0;
+    }, 500);
+  } else {
+    console.log("double click!");
+  }
+}
```
Comment (dinythomas89) [inline javascript/javascript2/week3/Warmup/warmUp.js]: https://developer.mozilla.org/en-US/docs/Web/API/Element/dblclick_event

**Review by dinythomas89 — APPROVED** [review]
Great. Hope now you have a better understanding of DOM, callbacks, objects and array

**Comment by hamoudi-hamoudi** [comment]
Yes I do . Thank you

File: javascript/javascript2/week3/Warmup/warmUp.js (hunk 1 above)
Comment (hamoudi-hamoudi) [reply javascript/javascript2/week3/Warmup/warmUp.js]: wow, this is much simpler :rolling_on_the_floor_laughing:

**Pull request merged**